app/views/layouts/application.html.haml

```diff
@@ -8,20 +8,32 @@
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
     = favicon_link_tag '/favicon.ico'
   %body
-    %nav.navbar.navbar-full.navbar-dark.bg-inverse
-      = link_to 'Bike Index Serial Search Inspector', root_path, class: 'navbar-brand'
-      %ul.nav.navbar-nav
-        %li.nav-item
-          = link_to 'Serial Searches', serial_searches_path, class: 'nav-link'
-        %li.nav-item
-          = link_to 'IP Addresses', ip_addresses_path, class: 'nav-link'
-        %li.nav-item
-          = link_to 'Graphs', graphs_path, class: 'nav-link'
+    - if current_user.superuser?
+      %nav.navbar.navbar-full.navbar-dark.bg-inverse
+        = link_to 'Admin', root_path, class: 'navbar-brand'
+        %ul.nav.navbar-nav
+          %li.nav-item
+            = link_to 'Users', accounts_path, class: 'nav-link'
+          %li.nav-item
+            = link_to 'Serial Searches', serial_searches_path, class: 'nav-link'
+          %li.nav-item
+            = link_to 'IP Addresses', ip_addresses_path, class: 'nav-link'
+          %li.nav-item
+            = link_to 'Graphs', graphs_path, class: 'nav-link'
+
+    - if notice.present?
+      %p.notice.alert-info
+        = notice
+    - if alert.present?
+      %p.alert.alert-danger
+        = alert
 
     = yield
 
     %footer.footer
       .container
+        = link_to 'Refresh Bike Index bikes', user_bike_index_omniauth_authorize_path
+        // Unsure about the path here. Coming back to that.
```

Comment:
Not entirely sure. I suppose once I put the job to request bike index bikes with the login route it would work. yes?